lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
@shalkam Sure, let me elaborate more:
As I explained in  https://github.com/strongloop/loopback-connector-mongodb/pull/301#discussion_r89230766

When we do CRUD operations of a model we can provide config in each function call:
 ```js
User.find({where: {account.amount: 1000}}, {mongodbAggregateConfig: {// whatever config you need}}, function(err, result){});
```